src/libnnstreamer-edge/nnstreamer-edge-internal.c

```diff
@@ -1258,20 +1259,29 @@ nns_edge_create_handle (const char *id, nns_edge_connect_type_e connect_type,
   eh->sending = false;
   eh->listener_fd = -1;
   eh->caps_str = nns_edge_strdup ("");
-  nns_edge_metadata_create (&eh->metadata);
+
+  ret = nns_edge_metadata_create (&eh->metadata);
+  if (ret != NNS_EDGE_ERROR_NONE) {
+    nns_edge_loge ("Failed to create edge metadata");
+    goto error;
+  }
+
   nns_edge_queue_create (&eh->send_queue);
 
   if (NNS_EDGE_CONNECT_TYPE_AITT == connect_type) {
     int ret = nns_edge_aitt_create (&eh->broker_h);
```

Comment:
let's remove declaration here, (int ret = ...)
`ret = nns_edge_aitt_create (&eh->broker_h);`

Reply:
I removed 'int', Thanks for your review!